libs/sdk-ui-ext/src/dashboardView/DashboardView.tsx

```diff
@@ -48,6 +48,10 @@ export const DashboardView: React.FC<IDashboardViewProps> = ({
     execConfig,
 }) => {
     const dashboardRef = typeof dashboard === "string" ? idRef(dashboard, "analyticalDashboard") : dashboard;
+    const defaultEmptyLayout: IDashboardLayout = {
```

Comment:
We should probably put this outside of the function body so that it is not recreated on every render. That might improve performance by avoiding some hook dependency changes.